src/Apis.jsx

```diff
@@ -0,0 +1,31 @@
+import { useState, useEffect } from "react";
+
+const useFlightApi = (url, id = null) => {
+  const [data, setData] = useState([]);
+  const [loading, setLoading] = useState(true);
+  const [error, setError] = useState(null);
+
+  useEffect(() => {
+    const fetchData = async () => {
+      try {
+        const response = await fetch(url);
+        if (!response.ok) {
+          throw new Error("Network response was not ok");
```

Comment:
This shouldn't be the error message. it should be something more end user friendly. 

"There was an error reaching to our server. Please try again later"